database/migrations/2024_04_19_091339_create_case_studies_table.php

```diff
@@ -13,7 +13,7 @@ public function up(): void
             $table->id();
             $table->string("photo");
             $table->string("company");
-            $table->string("slug");
+            $table->json("slug");
```

Comment:
Lest create new one migration for that update action :)

Reply:
created